src/main/java/org/sonar/plugins/stash/StashPlugin.java

```diff
@@ -41,7 +41,7 @@
   public static final String STASH_RESET_COMMENTS = "sonar.stash.comments.reset";
   public static final String STASH_URL = "sonar.stash.url";
   public static final String STASH_LOGIN = "sonar.stash.login";
-  public static final String STASH_PASSWORD = "sonar.stash.password";
+  public static final String STASH_PASS_PROPERTY = "sonar.stash.password";

```

Comment:
Is this because we should not use variables containing "password"? If this is the case it'd like to keep it as it is, as it's more clear.